pandas/_libs/src/parser/io.c

```diff
@@ -17,7 +17,7 @@ The full license is in the LICENSE file, distributed with this software.
 #define O_BINARY 0
 #endif  // O_BINARY
 
-#if PY_VERSION_HEX >= 0x03060000 && defined(_WIN32)
+#ifdef(_WIN32)
```

Comment:
I saw some compilation failures around this.

Can't we just do a pure clean and remove the `PY_VERSiON_HEX` from that if statement?